beesdoo_website_shift/views/shift_website_templates.xml

```diff
@@ -6,7 +6,7 @@
 <odoo>
 
     <!-- Add menu entries -->
-    <data noupdate="1">
+    <data noupdate="0">
```

Comment:
when it's 0 you can remove noupdate parameter. I think that you can even remove data tags as it's no more needed on 12.0 if not wanting to have noupdate set to True